libc/src/__support/time/gpu/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_object_library(
+  time_utils
+  SRCS
+    time_utils.cpp
+  HDRS
+    time_utils.h
+  DEPENDS
+    libc.hdr.types.clock_t
+    libc.hdr.time_macros
+)
+
+add_entrypoint_object(
```

Comment:
I really don't think it makes sense to have an entry point implementation live in `__support` Can we just make this an object library?

Reply:
You are right. I `ctrl+c/ctrl+v`ed it from original place. Changing it now

Reply:
fixed in https://github.com/llvm/llvm-project/pull/118886/commits/ea35f2a0cdc3b02d3299e49204d7b7fdd58a5456